networkx/algorithms/centrality/tests/test_current_flow_closeness.py

```diff
@@ -38,6 +38,32 @@ def test_current_flow_closeness_centrality_not_connected(self):
         with pytest.raises(nx.NetworkXError):
             nx.current_flow_closeness_centrality(G)
 
+    def test_current_flow_closeness_centrality_fewer_than_3_vertices(self):
+        G = nx.path_graph(2)
+        with pytest.raises(nx.NetworkXError):
+            nx.current_flow_closeness_centrality(G)
+
+    def test_current_flow_closeness_centrality_not_implemented_for(self):
+        G = nx.DiGraph()
+        G.add_edge(0, 1)
+        with pytest.raises(nx.NetworkXNotImplemented):
+            nx.current_flow_closeness_centrality(G)
+
+        G = nx.MultiGraph()
+        G.add_edges_from([(0, 1), (0, 1)])
+        with pytest.raises(nx.NetworkXNotImplemented):
+            nx.current_flow_closeness_centrality(G)
+
+        G = nx.MultiDiGraph()
+        G.add_edges_from([(0, 1), (0, 1)])
+        with pytest.raises(nx.NetworkXNotImplemented):
+            nx.current_flow_closeness_centrality(G)
```

Comment:
here, you can use parameterized tests, like this:
```suggestion
    @pytest.mark.parametrize("G", (nx.DiGraph(), nx.MultiGraph(), nx.MultiDiGraph()))
    def test_current_flow_closeness_centrality_not_implemented_for(self):
        with pytest.raises(nx.NetworkXNotImplemented):
            nx.current_flow_closeness_centrality(G)
```

Reply:
I was not aware of this feature, but that does look a lot cleaner. I can't remember why I thought I needed to explicitly add edges, though.